pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -444,7 +450,8 @@ func (c *Controller) onServiceEvent(curr interface{}, event model.Event) error {
 
 	log.Debugf("Handle event %s for service %s in namespace %s", event, svc.Name, svc.Namespace)
 
-	svcConv := kube.ConvertService(*svc, c.opts.DomainSuffix, c.Cluster())
+	svcConv := kube.ConvertService(*svc, c.opts.DomainSuffix, c.Cluster(),
+		c.imports.GetClusterSetIPs(kube.NamespacedNameForK8sObject(svc)))
```

Comment:
What happens when serviceImport does not exist when running this?

Reply:
It will get a `nil` array.